### PR TITLE
add getAppSettings and update questConfig loading method

### PR DESCRIPTION
**Pull request by JoeArmani**
Related: https://github.com/episphere/connect/issues/1048

-add `getAppSettings` function
-add Firestore `appSettings` collection
-update Quest version fetching process 

NOTE: Merge with ConnectFaas https://github.com/episphere/connectFaas/pull/612

### CODE REVIEW

**Review by amber-emmes — APPROVED** [review]
Looks good!

File: js/shared.js [hunk 1]
```diff
@@ -2003,4 +2003,41 @@ export const getSelectedLanguage = () => {
...
+            },
+        });
+    
+        if (!response.ok) {
+            throw new Error(`Server responded with: ${response.status}`);
+        }
+    
+        const jsonResponse = await response.json();
+        
+        if (jsonResponse.code === 200) {
```
Comment (anthonypetersen) [inline js/shared.js:2035]: Are the checks on lines 2029 and 2035 redundant? It doesn't look like the endpoint you created would return a code between 201-209.

Comment (JoeArmani) [reply js/shared.js]: Good point -- we really only need to check the `.code === 200`. Updated.

Comment (JoeArmani) [reply js/shared.js]: Note: I also just updated the Error syntax

**Review by anthonypetersen — APPROVED** [review]

**Pull request merged**